### ISSUE TITLE
Idea for new API

### ISSUE THREAD

**Issue by amazinigmech2418** [ateesdalejr/maildiscuss#8]
I think we should make an API page for users. It would allow for more advanced code including private messages. Also, profile pics could be added too. We could made the format like this:
`{
"username": "username",
"logo":"profilepic.png",
"messagenum":0
}`
We could also add more API, but this is just the idea.

**Comment by ateesdalejr** [comment]
I agree. Currently the only api available for users is `/api/get_bio.php?name=name`.

**Comment by amazinigmech2418** [comment]
What problems?

**Comment by ateesdalejr** [comment]
@amazinigmech2418 The current problems that we are having implementing asynchronous forum submissions.

**Comment by amazinigmech2418** [comment]
So will it use JS or PHP. It depends. If JS, I can help, but not PHP.

**Comment by ateesdalejr** [comment]
It will use JS I need to set jquery up to get it working.

**Comment by amazinigmech2418** [comment]
OK. <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script> is the code needed.

**Comment by amazinigmech2418** [comment]
I added jQuery.

**Comment by amazinigmech2418** [comment]
We should probably use cookies to stay logged in.

**Comment by csf30816** [comment]
Yes a session cookie that encodes the users username and password. Although I thought we already had that feature 👍

**Comment by amazinigmech2418** [comment]
OK. So how can I see what we already have? What's the URL for the website?

**Comment by csf30816** [comment]
Well the code is shown here so you could look but our testing URL is currently http://andrewjr.frvp.cf/ running Senior plan on FreeVP.

**Comment by amazinigmech2418** [comment]
OK.

**Comment by ateesdalejr** [comment]
@amazinigmech2418 If you want to find all API endpoints you can look in the API directory. The current login system is broken because we are still working on it. If I were to revert the testing env back to the old version it would still login.

**Comment by ateesdalejr** [comment]
Also the test server is running on the latest stable version 65fa956041b7d3580dcf0c2144bc1deb232c5952 so logging in should work well.

**Comment by csf30816** [comment]
I forgot my password, also do we encode the passwords?

**Comment by ateesdalejr** [comment]
Apologies when I reverted to stable I may have deleted your account.

**Comment by csf30816** [comment]
Hahaha no biggy I had an incorrect username anyways, I will just recreate the account :)

**Comment by csf30816** [comment]
There was an aJAX error I think :(

**Comment by ateesdalejr** [comment]
Do a hard refresh on the page. It may take a while for the revert to appear.
